src/App/Pages/Vault/GroupingsPage/GroupingsPageViewModel.cs

```diff
@@ -564,6 +642,42 @@ private async Task LoadDataAsync()
             }
         }
 
+        private bool IsVaultFilterMyVault => 
+            _vaultFilterSelection != null && _vaultFilterSelection == AppResources.MyVault;
+
+        private bool IsVaultFilterOrgVault =>
+            _vaultFilterSelection != null && _vaultFilterSelection != AppResources.AllVaults &&
+            _vaultFilterSelection != AppResources.MyVault;
+
+        private async Task<string> GetVaultFilterOrgIdAsync()
+        {
+            foreach (var org in _organizations)
+            {
+                if (org.Name == _vaultFilterSelection)
+                {
+                    return org.Id;
+                }
+            }
+            return null;
```

Comment:
This is the same as `return _organizations.FirstOrDefault(o => o.Name == _vaultFilterSelection)?.Id;`